contrib/pyln-spec/bolt7/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "pyln-bolt7"
+version = "1.0.186"
+description = "BOLT7"
+authors = ["Rusty Russell"]
+license = "BSD-MIT"
+
+packages = [
+  { include = "pyln/spec/bolt7" },
+]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+pyln-proto = { path = "../../pyln-proto" }
+
+[tool.poetry.dev-dependencies]
+
```

Comment:
maybe we can use the power of poetry dependencies to add also some dev dep `poetry add pytest --dev`?

Reply:
Mh maybe this one https://github.com/ElementsProject/lightning/blob/master/contrib/pyln-spec/bolt7/tests/test_bolt7.py  or it is only a mock test?